plone/namedfile/scaling.py

```diff
@@ -17,7 +17,12 @@
 from zope.traversing.interfaces import ITraversable, TraversalError
 from zope.publisher.interfaces import IPublishTraverse, NotFound
 from zope.app.file.file import FileChunk
-from plone.protect.interfaces import IDisableCSRFProtection
+
+try:
+    # Soft dependency to make this package work without plone.protect
+    from plone.protect.interfaces import IDisableCSRFProtection
+except ImportError:
```

Comment:
That would be this one: https://github.com/plone/plone.api/issues/217
which links to this: https://github.com/plone/plone.app.event/commit/a17c9e1178303639893a95141b468a73a654b5f5